lib/airtable/table.rb

```diff
@@ -71,8 +93,8 @@ def result_attributes(res)
     end
 
     def worksheet_url
-      "/#{app_token}/#{URI.encode(worksheet_name)}"
+      "/#{app_token}/#{CGI.escape(worksheet_name)}"
```

Comment:
well, `URI.encode` is deprecated and was throwing warnings - https://github.com/ruby/ruby/commit/238b979f1789f95262a267d8df6239806f2859cc

`CGI.escape` will convert spaces to `+` instead of `20`. I guess I should have checked the rules for naming tables.